### ISSUE TITLE
YouTube link for new singles

### ISSUE THREAD

**Issue by oxidand** [alexkay/muspy#51]
It would be great to have a link to YouTube (e.g. <https://www.youtube.com/results?search_query=%s>) for the singles. Possible?

**Comment by mrx23dot** [comment]
yes please,  a direct link to Youtube search for the title (for any kind of release), in RSS+email
worst case nothing comes up.
